src/index.js

```diff
@@ -1,3 +1,19 @@
-const sum = (c, d) => c + d;
+/* eslint-disable */
 
-export default sum;
+import * as CalendarUtils from './utils/CalendarUtils/CalendarUtils';
+import * as Proxy from './utils/Proxy/Proxy';
+
+const TeamContributionCalendar = (
+  container,
+  gitHubUsers,
+  gitLabUsers = [],
+  proxyUrl = Proxy.defaultProxyServerUrl,
+) => {
+  if (CalendarUtils.RequiredParamsExist(container, gitHubUsers)) {
+    console.info('Params are OK.');
+  } else {
+    console.error('You must provide the required parameters in the appropriate format.');
```

Comment:
Here you can throw an exception already.

Reply:
Cool, okay!